packages/plugin-board/src/realtime.js

```diff
@@ -82,9 +82,47 @@ const RealtimeService = Mercury.extend({
     if (contentType) {
       data.data.contentType = contentType;
     }
+
+    // use mercury socket if it is shared
+    // if this.socket is defined then use it instead
+    const isSharingMercuryFeatureEnabled = this.spark.device.features.developer.get(`web-shared-mercury`);
+    if (isSharingMercuryFeatureEnabled && isSharingMercuryFeatureEnabled.value && !this.socket) {
+      return this.spark.mercury.socket.send(data);
+    }
+
     return this.socket.send(data);
   },
 
+  /**
+   * Connect and use an exisiting mercury connection
+   * @memberof Board.RealtimeService
+   * @param  {Board~Channel} channel
+   * @returns {Promise<Board~Registration>}
+   */
+  connectToSharedMercury(channel) {
```

Comment:
We make this call first when we want to share mercury connection (usually when we open a new whiteboard), then the response will include `sharedWebSocket`:
```JSON
{
   "mercuryConnectionServiceClusterUrl":"...",
   "binding":"...",
   "webSocketUrl":"...",
   "sharedWebSocket":true,
   "action":"REPLACE"
}
```
If `sharedWebSocket` is true, we won't need to do anything as board binding is registered.

If `sharedWebSocket` is false, then we will need to open a second web socket using `webSocketUrl`. I am letting the web-client handle this situation. Do you think it's better to have this logic in the sdk?